test/unit/math/prim/mat/fun/mdivide_left_tri_test.cpp

```diff
@@ -1,6 +1,18 @@
 #include <stan/math/prim/mat.hpp>
 #include <gtest/gtest.h>
 
+#ifdef STAN_OPENCL
+#include <stan/math/opencl/opencl_context.hpp>
+#include <stan/math/opencl/multiply.hpp>
+#include <stan/math/opencl/copy.hpp>
+#include <stan/math/opencl/tri_inverse.hpp>
+#include <boost/random/mersenne_twister.hpp>
```

Comment:
There is a mersenne in the standard library

http://www.cplusplus.com/reference/random/mt19937/

Reply:
The mersenne twister generator from `Boost` is used in most tests that need randomness and the boost generators are used in the `stan::math::uniform_rng` so I think we should stick with the Boost one here.

 If you feel strongly about this I can change it tho.

Reply:
Nah that's fine just checking!